core/controllers/pages.py

```diff
@@ -40,6 +42,11 @@
     'site_forum_url', {'type': 'unicode'},
     'The site forum URL (for links; the Forum page is configured separately)',
     default_value='https://site/forum/url')
+MODERATOR_REQUEST_FORUM_URL = config_domain.ConfigProperty(
+    'moderator_request_forum_url', {'type': 'unicode'},
+    'A link to the forum for nominating explorations to be featured '
```

Comment:
Understood. I would add a note that the participation playbook is the only place this is used.